src/frontend/src/pages/CreateWorkPackagePage/CreateWPFormView.tsx

```diff
@@ -11,167 +11,223 @@ import Typography from '@mui/material/Typography';
+              render={({ field: { onChange, value } }) => (
+                <>
+                  <Box>
+                    <Typography variant="caption">Start Date (YYYY-MM-DD)</Typography>
+                  </Box>
+                  <DatePicker
+                    inputFormat="yyyy-MM-dd"
+                    onChange={onChange}
+                    className={'padding: 10'}
+                    value={value}
+                    renderInput={(params) => <TextField autoComplete="off" {...params} />}
+                  />
+                </>
+              )}
+            />
+          </Grid>
+          <Grid item xs={2}>
+            <Box>
+              <Typography variant="caption">Duration</Typography>
+            </Box>
+            <ReactHookTextField
+              name="duration"
+              control={control}
+              placeholder="Enter duration..."
+              errorMessage={errors.duration}
+              type="number"
+              endAdornment={<InputAdornment position="end">weeks</InputAdornment>}
+              sx={style}
+            />
+          </Grid>
+          <Grid item xs={12}>
+            <Box marginBottom={1}>
+              <Typography variant="caption">Dependencies</Typography>
+            </Box>
+            {dependencies.map((_element, i) => {
+              return (
+                <Grid item sx={{ display: 'flex', alignItems: 'center' }}>
+                  <TextField required autoComplete="off" {...register(`dependencies.${i}.wbsNum`)} sx={{ width: 1 / 10 }} />
+                  <IconButton type="button" onClick={() => removeDependency(i)} sx={{ mx: 1, my: 0 }}>
+                    <DeleteIcon />
+                  </IconButton>
+                </Grid>
+              );
+            })}
+            <Button variant="contained" color="success" onClick={() => appendDependency({ wbsNum: '' })} sx={{ my: 2 }}>
+              + ADD NEW DEPENDENCY
             </Button>
-          </Box>
-        </form>
+            <Box marginBottom={1}>
+              <Typography variant="caption">Expected Activities</Typography>
+            </Box>
+            <ReactHookEditableList
+              name="expectedActivities"
+              register={register}
+              ls={expectedActivities}
+              append={appendExpectedActivity}
+              remove={removeExpectedActivity}
+            />
+            <Box marginBottom={1}>
+              <Typography variant="caption">Deliverables</Typography>
+            </Box>
+            <ReactHookEditableList
+              name="deliverables"
+              register={register}
+              ls={deliverables}
+              append={appendDeliverable}
+              remove={removeDeliverable}
+            />
+          </Grid>
+        </Grid>
+        <Box display="flex" flexDirection="row-reverse" gap={2}>
+          <Button variant="contained" color="primary" type="submit" disabled={!allowSubmit}>
+            Create
```

Comment:
same button problem as other form, also the position of the create/cancel buttons are flipped on this page and are on the right side instead of the left, idrk which one we go with, whichever looks better to you for now tbh (might be worth pinging james to ask about), but whatever you decide on pls make it standard for both forms

Reply:
The form is giving me an error "WBS Invalid: WBS #s include periods, none found" when I try to submit with project wbsNum 1.1.0